x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -120,7 +120,7 @@ export interface AddAuditEventParams {
    * Relevant saved object information
    * object containing type & id strings
    */
-  savedObject?: { type: string; id: string };
+  savedObject?: { type: string; id: string; name?: string };
```

Comment:
Self Review: This was not totally in scope of the change, but it was required for the testing